zboxcore/sdk/listworker.go

```diff
@@ -143,15 +143,22 @@ func (req *ListRequest) getlistFromBlobbers() []*listResponse {
 	return listInfos
 }
 
-func (req *ListRequest) GetListFromBlobbers() *ListResult {
+func (req *ListRequest) GetListFromBlobbers() (*ListResult, error) {
 	lR := req.getlistFromBlobbers()
 	result := &ListResult{}
 	selected := make(map[string]*ListResult)
 	childResultMap := make(map[string]*ListResult)
+	var err error
+	var errNum int
 	for i := 0; i < len(lR); i++ {
 		req.consensus = 0
 		ti := lR[i]
-		if ti.err != nil || ti.ref == nil {
+		if ti.err != nil {
+			err = ti.err
+			errNum++
+			continue
```

Comment:
if we see the error, why not return error there itself?

Reply:
@boddumanohar  list should work even 1 blobber fails.